Content.Shared/Implants/SharedSubdermalImplantSystem.cs

```diff
@@ -62,7 +63,7 @@ private void OnRemove(EntityUid uid, SubdermalImplantComponent component, EntGot
         if (component.ImplantAction != null)
             _actionsSystem.RemoveProvidedActions(component.ImplantedEntity.Value, uid);
 
-        if (!_container.TryGetContainer(uid, BaseStorageId, out var storageImplant))
+        if (!_container.TryGetContainer(uid, BaseStorageId, out var storageImplant) || Terminating(storageImplant.ContainedEntities.FirstOrDefault()))
```

Comment:
I think it should just iterate each one and dump it if it's not deleting as this is still be subject to the same issue, just less frequently.

Reply:
Thought that might've been slightly slower, but it's not gong to ever be a ton of items so it won't matter too much. I also moved the TODO note to the containersystem now.